packages/vite-plugin-cloudflare/src/module-fallback.ts

```diff
@@ -408,3 +423,22 @@ function isCommonJS(code: string): boolean {
 
 	return false;
 }
+
+async function bundleDependency(specifier: string) {
+	// Can't use the standard `import.meta.resolve()` here.
+	// That function doesn't exist when run in `vm.runInThisContext()`
+	// i.e. inside a Vitest test.
+	const entryFile = fileURLToPath(
+		importMetaResolve(specifier, import.meta.url),
+	);
+	const transpiled = await build({
+		entryPoints: [entryFile],
+		bundle: true,
+		write: false,
+		format: 'cjs',
```

Comment:
Just for my understanding, why is it we bundle as cjs here?

Reply:
In order for things like `require("node:buffer")` to work, I needed the module being uploaded to be of type `NodeJsCompatModule`, which required the code to be compiled to CommonJS.